scripts/publish_python.js

```diff
@@ -8,125 +8,258 @@
  */
 
 const {execute} = require("./script_utils.js");
-const fs = require("fs");
```

Comment:
This is great but I need this script until we are certain GHA stabilizes (after this PR is merged).  Can we please also move this to the "GHA stabilization PR".